File: system/testSoluttions/battleshipGame/template.pas
program pas;
var
    firstAction: string;
    row: integer;
    column: integer;
    answer: integer;
    response: string;
begin
    //Запишите сюда первый ход в формате "строка-столбец"
    firstAction := '';
    writeln(firstAction);

    while (true) do
    begin
        //Ответ на выстрел
        readln(answer);
        //Заполните переменные row и column

        response := row + '-' + column;
        writeln(response);
    end;
end.

File: system/testSoluttions/battleshipGame/template.c
#include <stdio.h>

int main() {
    int answer;
    int row;
    int column;

    //Запишите сюда первый ход в формате "строка-столбец"
    char firstAction[] = "\n";
    fprintf(stdout, firstAction);
    fflush(stdout);

    while (1) {
        //Ответ на выстрел
        scanf("%d", &answer);
        //Заполните переменные row и column


        fprintf(stdout, "%d-%d\n", row, column);
        fflush(stdout);
    }
    return 0;
}
